src/@next/globalStyles/global.ts

```diff
@@ -0,0 +1,49 @@
+import { createGlobalStyle } from "styled-components";
+import { DefaultTheme } from ".";
+
+export const GlobalStyle = createGlobalStyle<{ theme: DefaultTheme }>`
+  body {
+    font-family: ${props => props.theme.typography.baseFontFamily};
+    font-size: ${props => props.theme.typography.baseFontSize};
+    line-height: ${props => props.theme.typography.baseLineHeight};
+    color: ${props => props.theme.colors.baseFontColor};
+
```

Comment:
why things below are nested in body?

Reply:
Both options would seem to work, as everything is scoped inside `body`. I copied this from SCSS file and didn't pay much attention to it 🙊. Is there a reason why it should not be inside `body`?

Reply:
I think it's a good practice to not nest elements if it's not necessary. It won't change anything, but it will look a bit better. 
I haven't noticed it before too....